src/main/java/pb360/controller/GenerateRestApiController.java

```diff
@@ -0,0 +1,50 @@
+package pb360.controller;
+
+import pb360.model.MessageObject;
+import pb360.data.repository.MessageObjectRepository;
+
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("v1/controllers")
+public class GenerateRestApiController {
+
+    @Autowired
+    MessageObjectRepository messageObjectRepository;
+
+    @RequestMapping(method=RequestMethod.GET, value="v1/controllers")
```

Comment:
please remove value, because right url is: "/v1/controllers" for GET ALL/POST not "v1/controllersv1/controllers" this url will be compile when you run the project